Dockerfile

```diff
@@ -12,7 +12,7 @@ WORKDIR /code
 COPY . /code
 
 # Run bundle install to install the Ruby dependencies.
-RUN gem install bundler
+RUN gem install bundler -v 2.3.26
```

Comment:
I can't remember exactly why this was needed. I think the version of Ruby we use it too far out of date. Gotta do #61.